src/test/ui/parser/issue-88276-unary-plus.rs

```diff
@@ -0,0 +1,13 @@
+// run-rustfix
+#[allow(unused_parens)]
+fn main() {
+    let _ = +1; //~ ERROR leading `+` is not supported
+    let _ = -+(1+2)*3; //~ ERROR leading `+` is not supported
+    let _ = -+-+(1+2)*3; //~ ERROR leading `+` is not supported
+                         //~| ERROR leading `+` is not supported
+    let _ = (1 + +2) * +3; //~ ERROR leading `+` is not supported
+                           //~| ERROR leading `+` is not supported
+    let _ = (+&"hello"); //~ ERROR leading `+` is not supported
```

Comment:
So this is one case where removing the `+` might be obvious when written like this, but I don't know about the general case.